chore: remove PanoramaView, PanoramaLayer, StaticSource and PanoramaTileBuilder

gchoqueux opened the pull request:
Replaced by the feature : **Immersive and OrientedImage.**

### BREAKING CHANGE:

Remove `PanoramaView, PanoramaLayer, StaticSource and PanoramaTileBuilder.`

fix #1048

zarov reviewed (approved):
Thanks ! You forgot `PanoramaLayer` in docs/navigation.json` and that would be good.

gchoqueux committed:
chore: remove PanoramaView, PanoramaLayer, StaticSource and PanoramaTileBuilder.

Replaced by the feature : Immersive and OrientedImage.

BREAKING CHANGE:

Remove PanoramaView, PanoramaLayer, StaticSource and PanoramaTileBuilder.
